docs/pydantic.md

```diff
@@ -17,7 +17,7 @@ from dataclasses_avroschema.pydantic import AvroBaseModel
 from pydantic import Field
 
 
-class FavoriteColor(str, enum.Enum):
+class FavoriteColor(str, str, enum.Enum):
```

Comment:
One `str` should be removed. The same for the other cases.

Reply:
Those are typos, thanks for the heads up!